packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp

```diff
@@ -141,6 +141,9 @@ setParameters (const Teuchos::ParameterList& List)
     if (relaxType == "Jacobi") {
       PrecType_ = Ifpack2::Details::JACOBI;
     }
+    else if (relaxType == "MTJacobi") {
+      PrecType_ = Ifpack2::Details::MTJACOBI;
+    }
```

Comment:
Would it make sense for us to call this "with coloring" instead?  If we were to use (say) HTS to implement Gauss-Seidel, then it would be multithreaded too.  The enums don't have to change but the Doxygen and ParameterList documentation could.